Dockerfile

```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
     rm -fr /tmp/sqliteodbc-0.9996 && \
     apt-get purge -y build-essential sqlite3 libsqlite3-dev unixodbc-dev && \
     rm -rf /var/cache/apk/*
-    
+ 
+# Install Python, pip and requests library 
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
```

Comment:
Does it really need python3-dev - shouldn't python3 be sufficient?

Reply:
Good point. I tested with python3 and it seems to be working. I will make this change.